tests/robottelo/test_issue_handlers.py

```diff
@@ -1,421 +0,0 @@
-from collections import defaultdict
```

Comment:
I see we're deleting the entire test module. It would be worthwhile to create a few tests for the Jira issue handler.

Reply:
created https://github.com/SatelliteQE/robottelo/issues/19012